src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -54,14 +54,17 @@ protected string ParseSecretToken(ConfigurationKeyValue kv)
 		protected bool ParseCaptureHeaders(ConfigurationKeyValue kv)
 		{
 			if (kv == null || string.IsNullOrEmpty(kv.Value)) return true;
-			if (bool.TryParse(kv.Value, out var value)) return value;
 
-			return true;
+			return !bool.TryParse(kv.Value, out var value) || value;
 		}
 
 		protected LogLevel ParseLogLevel(ConfigurationKeyValue kv)
 		{
-			if (TryParseLogLevel(kv?.Value, out var level)) return level.Value;
+			if (TryParseLogLevel(kv?.Value, out var level))
+			{
+				if (level != null)
```

Comment:
That's all true, but it still causes a warning:

<img width="1149" alt="image" src="https://user-images.githubusercontent.com/1091853/61326779-d2735a00-a817-11e9-9250-5bd5070d05e2.png">

I think this is a really minor change and fine this way - I just wanted to get rid of the warning.

Reply:
The warning is indeed relevant because from `out var level` `level` is deduced to be `LogLevel?` and indeed `Nullable<T>.Value` can throw `InvalidOperationException`. I think the better solution to this issue would be to make `level` parameter of `TryParseLogLevel` non-nullable as I suggested above.

Not a major point - you decide.